graph/loaders/comments.js

```diff
@@ -218,6 +218,14 @@ const getCommentCountByQuery = (context, {ids, statuses, asset_id, parent_id, au
     query = query.where({author_id});
   }
 
+  if (tag) {
+    query = query.find({
+      'tags.tag.name': {
+        $in: tag,
```

Comment:
Yeah my first thought was `tags` too. Someone already added that to the definitions (without implementing the loader) to the graphql and used `tag` so I thought maybe it had some good reasoning :-)